plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterAttribute.php

```diff
@@ -28,7 +28,6 @@ final class ProductFilterAttribute extends AbstractBlock {
 	protected function initialize() {
 		parent::initialize();
 
-		add_filter( 'woocommerce_blocks_product_filters_param_keys', array( $this, 'get_filter_query_param_keys' ), 10, 2 );
```

Comment:
Could this be considered a breaking change? I’m okay with proceeding, but we should probably publish a developer advisory at the very least. What do you think?